package-lock.json

```diff
@@ -35401,7 +35413,7 @@
       }
     },
     "pandora-book": {
-      "version": "file:tools/pandora-book-3.0.0-66.tgz",
+      "version": "file:tools\\pandora-book-3.0.0-66.tgz",
```

Comment:
Why it change the directory separater

Reply:
However, I think this should not change. I will restore this point back the base version

Reply:
@Sakchai-Refinitiv I already reverted this change back to the base version